_posts/0000-01-02-Akshay-RS.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
Mom - she is an embodiment of love.
She cares for us; she prays for us.
She scolds us; but she molds us.
She is what makes us who we are.